CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 [Next release](https://github.com/barryvdh/laravel-ide-helper/compare/v2.9.3...master)
```

Comment:
Actually, it was wrong the last time to bump this eagerly -> this not bumping now